src/main/webapp/front-end/src/utilities.js

```diff
@@ -1,35 +1,43 @@
 import store from './store.js';
+import Cookies from 'js-cookie';
 import { login, loginError, loginSuccess, loginClearInputs } from './actions.js';
 var _ = require('lodash');
 
-export var authenticatedSession = null;
+export var authenticationHeader = 'bee-session-key'
+export var cookieName = 'BEE_SESS_KEY'
 
-export var authenticationHeader = "bee-session-key"
+export function getSession() {
+  return Cookies.get(cookieName);
+}
 
 export function setSession(session) {
-    authenticatedSession = session
+  if (session == null) {
+    Cookies.remove(cookieName);
+  } else {
+    Cookies.set(cookieName, session, { expires: 30 });
+  }
```

Comment:
ok this seems fine for now.  the problem is of course that if the user has sessions disabled the whole thing doesn't work. 

a more robust approach is to have a variable 'authenticatedSession' that, only when it is null, checks the cookie to see if there is a non-null value.  basically use the cookie as a db, instead of as the js variable.  

obviously when editing the session you would change both the authenticatedSession variable and the cookie.